client/my-sites/stats/site.jsx

```diff
@@ -200,6 +201,23 @@ class StatsSite extends Component {
 		return isValid ? inputDate : null;
 	}
 
+	// Return a default amount of days to subtracts from the present day depending on the period selected.
+	// Used in case no starting date is present in the URL.
+	getDefaultRangeForPeriod( period ) {
```

Comment:
Maybe `getDefaultDaysForPeriod()`? My initial reading is that it's returning a "range" which I'd expect to be two values (or an object).